virtool/startup.py

```diff
@@ -286,8 +286,32 @@ async def startup_redis(app: typing.Union[dict, Application]):
     scheduler = get_scheduler_from_app(app)
     await scheduler.spawn(periodically_ping_redis(redis))
 
+    return redis
+
+
+async def startup_databases(app: Application):
+    """
+    Runs the three database startup coroutines concurrently
+    :param app: the app object
+
+    """
+    mongo_startup = asyncio.create_task(startup_db(app))
+    pg_startup = asyncio.create_task(startup_postgres(app))
+    redis_startup = asyncio.create_task(startup_redis(app))
+
+    mongo, pg, redis = await asyncio.gather(mongo_startup, pg_startup, redis_startup)
```

Comment:
```suggestion
    mongo, pg, redis = await asyncio.gather(
      startup_db(app), 
      startup_postgres(app),
      startup_redis(app)
    )
```
Except the called functions should be `connect_some_service()`.